plugins/org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/ui/AbstractAutoEditTest.java

```diff
@@ -37,6 +39,8 @@
 	public void setUp() throws Exception {
 		super.setUp();
 		closeWelcomePage();
+		IProject project = createProject("foo");
```

Comment:
I'm afraid this will dramatically increase the test runtime for all of our auto edit tests. I'd rather avoid this in the base class.